CardinalityEstimation.Test/CardinalityEstimatorTests.cs

```diff
@@ -65,6 +65,25 @@ public void TestGetSigma()
         }
```

Comment:
The point of serializedDense_v1.0.bin, serializedDirect_v1.0.bin and serializedSparse_v1.0.bin is to make sure we don't introduce breaking changes without incrementing the major version - these files should not be changed :)  Rather, we need to support backwards compatibility.

As I mentioned, I already have a serializer change in the works (in the Murmur3 branch), so you can just drop the serializer change here, and I'll make sure the CountElementsAdded is serialized as well in my next merge.

Thanks,
Oron